alerting/src/main/kotlin/org/opensearch/alerting/util/ClusterMetricsCoordinator.kt

```diff
@@ -0,0 +1,386 @@
+            ClusterMetricsDataPoint(
+                ClusterMetricsDataPoint.MetricType.NUMBER_OF_NODES,
+                currentTime,
+                numNodes
+            ),
+            ClusterMetricsDataPoint(
+                ClusterMetricsDataPoint.MetricType.NUMBER_OF_DATA_NODES,
+                currentTime,
+                numDataNodes
+            )
+        )
+
+//        val clusterstatusData = ClusterMetricsDataPoint(
+//            ClusterMetricsDataPoint.MetricType.CLUSTER_STATUS,
+//            currentTime,
+//            clusterStatus
+//        )
+//        val unassignedShardsData = ClusterMetricsDataPoint(
+//            ClusterMetricsDataPoint.MetricType.UNASSIGNED_SHARDS,
+//            currentTime,
+//            unassignedShards
+//        )
+//        val cpuUsageData = ClusterMetricsDataPoint(
+//            ClusterMetricsDataPoint.MetricType.CPU_USAGE,
+//            currentTime,
+//            avgCPU,
+//            minimumCPU,
+//            maximumCPU
+//        )
+//        val jvmDataPoint = ClusterMetricsDataPoint(
+//            ClusterMetricsDataPoint.MetricType.JVM_PRESSURE,
+//            currentTime,
+//            avgJVM,
+//            minimumJVM,
+//            maximumJVM
+//        )
+//        val pendingTasksData = ClusterMetricsDataPoint(
+//            ClusterMetricsDataPoint.MetricType.NUMBER_OF_PENDING_TASKS,
+//            currentTime,
+//            numPending
+//        )
+//        val activeShardsData = ClusterMetricsDataPoint(
+//            ClusterMetricsDataPoint.MetricType.ACTIVE_SHARDS,
+//            currentTime,
+//            activeShards
+//        )
+//        val relocatingShardsData = ClusterMetricsDataPoint(
+//            ClusterMetricsDataPoint.MetricType.RELOCATING_SHARDS,
+//            currentTime,
+//            relocatingShards
+//        )
+//        val nodesData = ClusterMetricsDataPoint(
+//            ClusterMetricsDataPoint.MetricType.NUMBER_OF_NODES,
+//            currentTime,
+//            numNodes
+//        )
+//        val dataNodesData = ClusterMetricsDataPoint(
+//            ClusterMetricsDataPoint.MetricType.NUMBER_OF_DATA_NODES,
+//            currentTime,
+//            numDataNodes
+//        )
+
+        dataPoints.forEach { clusterMetricsDataPoint ->
+            try {
+                val request = IndexRequest(ClusterMetricsVisualizationIndex.CLUSTER_METRIC_VISUALIZATION_INDEX)
+                    .source(
+                        clusterMetricsDataPoint.toXContent(
+                            XContentFactory.jsonBuilder(),
+                            ToXContent.MapParams(mapOf("with_type" to "true"))
+                        )
+                    )
+                val indexResponse: IndexResponse = client.suspendUntil { client.index(request, it) }
+                val failureReasons = checkShardsFailure(indexResponse)
+                if (failureReasons != null) {
+                    log.info("failed because $failureReasons")
```

Comment:
Perhaps use `log.error()` here instead of `log.info()`. Could potentially be refactored to something like:
```
log.error("Failed to index ${clusterMetricsDataPoint.type} data point.", failureReasons)